sample/src/main/java/com/adsbynimbus/android/sample/admanager/AdManagerFragment.kt

```diff
@@ -104,7 +104,9 @@ class AdManagerFragment : Fragment(), NimbusRequest.Interceptor {
                 it.addListener("Blocking Interstitial Controller")
             }
             AdItem.REWARDED_STATIC -> adManager.showRewardedAd(
-                NimbusRequest.forInterstitialAd("test_rewarded_static"),
+                NimbusRequest.forInterstitialAd("test_rewarded_static").apply {
```

Comment:
Is that a required change or is it optional?

Reply:
It's one way to force this to be a static ad, our interstitials are hybrid by default